src/sharepoint/rest/Queryable.ts

```diff
@@ -220,6 +220,22 @@ export class QueryableCollection extends Queryable {
         this._query.add("$select", selects.join(","));
         return this;
     }
+    public expand(...expands: string[]): QueryableCollection {
+        this._query.add("$expand", expands.join(","));
+        return this;
+    }
+    public orderby(...orderbys: string[]): QueryableCollection {
+        this._query.add("$orderby", orderbys.join(","));
+        return this;
+    }
+    public skip(skip: number): QueryableCollection {
```

Comment:
For Skip I did see [this](https://msdn.microsoft.com/en-us/library/office/dn531433.aspx#sectionSection1):

> The OData $skip query option does not work when querying items of a particular list. It does work when querying the lists endpoint.

So I am not sure adding skip to the queryable collection is correct - OR we need to do some overriding in the items collection. Not sure of the best way to handle this.